### DISCUSSION TITLE
Is there a way to list all tasks available even if no description is present

### DISCUSSION THREAD

**Discussion by apatniv** [go-task/task#452] [Q&A]
Currently, I can print a list of tasks if description is present. However, if description is not present, then task seems to emit an warning but don't print the list of tasks.

```bash
task -l
task: No tasks with description available
```

**Comment by andreynering** [comment]
As I said [here](https://github.com/go-task/task/issues/461#issuecomment-808903652), there's an open PR about that.

In the future, avoid creating multiple issues for the same feature request.

**Reply by apatniv** [reply]
I didn't know that there was an existing task. I searched but couldn't find it.
